src/requirements/requirement-frontend-computation.ts

```diff
@@ -12,22 +13,13 @@ type FulfillmentStatistics = {
   readonly courses: readonly (readonly CourseTaken[])[];
 } & RequirementFulfillmentStatistics;
 
-/**
- * @param course course object with useful information retrived from Cornell courses API.
- * @returns true if the course is AP/IB equivalent course or credit
- */
-const courseIsAPIB = (course: CourseTaken): boolean =>
-  [CREDITS_COURSE_ID, FWS_COURSE_ID].includes(course.courseId) ||
-  ['AP', 'IB'].includes(course.code.split(' ')[0]);
-
```

Comment:
I moved this function to `requirement-frontend-utils.ts` because it is also used there.